src/stan/lang/ast/node/row_vector_expr_def.hpp

```diff
@@ -0,0 +1,22 @@
+#ifndef STAN_LANG_AST_NODE_ROW_VECTOR_EXPR_DEF_HPP
+#define STAN_LANG_AST_NODE_ROW_VECTOR_EXPR_DEF_HPP
+
+#include <stan/lang/ast.hpp>
+#include <vector>
+
+namespace stan {
+  namespace lang {
+
+    row_vector_expr::row_vector_expr() : args_() { }
+
+    row_vector_expr::row_vector_expr(const std::vector<expression>& args)
+      : args_(args) { }
+
+    row_vector_expr& row_vector_expr::operator=(const row_vector_expr& ve) {
```

Comment:
removed `operator=()` from `array_expr`, `matrix_expr`, and `row_vector_expr`.   I see that `double_literal` and `int_literal` also have `operator=()` defined, and there too, all member variables are copied.  more cleanup?